src/validation.cpp

```diff
@@ -349,8 +349,9 @@ void CChainState::MaybeUpdateMempoolForReorg(
     while (it != disconnectpool.queuedTx.get<insertion_order>().rend()) {
         // ignore validation errors in resurrected transactions
         if (!fAddToMempool || (*it)->IsCoinBase() ||
-            AcceptToMemoryPool(
-                *this, *m_mempool, *it, true /* bypass_limits */).m_result_type !=
+            AcceptToMemoryPool(Params(),
```

Comment:
Consider using the global namespace operator `::Params()` to explicitly show that you're calling a global function. I think the long-term goal is to remove these global calls to make it easier to unit test.

Same comment for the ATMP calls below.

Reply:
Removed calls to global Params() in ATMP and in the tx_pool test file ( aac6548 )